### ISSUE TITLE
add command for retrieving plugin status

### ISSUE THREAD

**Issue by Klap-in** [dokufreaks/dokuwiki-translation#108]
the setting page shows some details, but no error messages
https://translate.dokuwiki.org/plugin/dw2pdf/settings

Add command that displays these errors. Store all error messages and not only latest. Seems the mailed message is stored as message, 250 characters is not enough.

Workaround: 
```
select name,state,url,branch,from_unixtime(lastUpdate),errorMsg,errorCount 
from repository 
where name='yourplugin';
```

**Comment by Klap-in** [comment]
ah, there is already a `dokuwiki:showStats`. Shows basic stuff for all extensions. Error output is still lacking. Also add filter for all extensions with number of error >0

**Comment by Klap-in** [comment]
`dokuwiki:showStats` is renamed with `dokuwiki:showInfo` and will show the entire error messages if a single repository is shown.
i.e.  `dokuwiki:showInfo plugin example` . Implemented in #112
